packages/lib-classifier/src/plugins/tasks/SurveyTask/mock-data/task.js

```diff
@@ -771,19 +771,19 @@ const task = {
       'label': 'What behaviors do you see?',
       'answers': {
         'TNG': {
-          'label': ' Eating'
+          'label': 'Eating'
```

Comment:
In option components, maybe should check if option label has a leading space, reformat to remove leading space, for backwards compatibility?

Reply:
Trimming should work for this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim

Reply:
Now I'm not sure why I removed the leading whitespace to begin with, might have been before I refactored the labels with the Grommet Text component. After adding back leading space to mock data, before refactoring with `trim()`, the radio and checkbox labels appear ok (without leading space)?

Reply:
![Screen Shot 2021-05-04 at 2 32 01 PM](https://user-images.githubusercontent.com/12118751/117059869-e4bcd180-ace5-11eb-9145-6aec39606106.png)
![Screen Shot 2021-05-04 at 2 31 55 PM](https://user-images.githubusercontent.com/12118751/117059884-e8e8ef00-ace5-11eb-86a2-e78947486095.png)